Src/components/dactive-card/DactiveCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { View, Text, ImageBackground } from 'react-native'
+import Button from '../dactive-card-buttons/DactiveCardButton'
+import { styles } from './dactive-card-style'
+
+interface IDActiveCardProp {
+  item: {
+    title: string
+    img: any
+  }
```

Comment:
in this case we create one more interface
interface IData {
 
    title: string
    img: any
  }
  
  interface IDActiveCardProp {
  item: IData
  color: string
}